renovate.json

```
@@ -0,0 +1,6 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "local>elastic/renovate-config"
```

To support the chainguard images only:

```suggestion
    "github>elastic/renovate-config:only-chainguard"
```